RubeesEat.IntegrationTests/CreateTestDb.sql

```diff
@@ -7,7 +7,8 @@ CREATE TABLE Persons
     PersonID VARCHAR(64) PRIMARY KEY,
     FirstName VARCHAR(64) NOT NULL,
     LastName VARCHAR(64) NOT NULL,
-    IsActive BOOLEAN NOT NULL DEFAULT TRUE
+    IsActive BOOLEAN NOT NULL DEFAULT TRUE,
+    LoginName VARCHAR(64)
```

Comment:
Don't know if you already did: please add a ticket that Person.LoginName must not be null in database or domain. This is a consequence of the login/inactive-user topic.